src/core/Elsa.Abstractions/Extensions/CompositeActivityBlueprintExtensions.cs

```diff
@@ -9,6 +9,19 @@ public static class CompositeActivityBlueprintExtensions
     {
         public static IEnumerable<IActivityBlueprint> GetEndActivities(this ICompositeActivityBlueprint workflowBlueprint) => workflowBlueprint.Activities.Where(x => !workflowBlueprint.GetOutboundConnections(x.Id).Any());
         public static IActivityBlueprint? GetActivity(this ICompositeActivityBlueprint workflowBlueprint, string id) => workflowBlueprint.Activities.FirstOrDefault(x => x.Id == id);
+        public static string? GetJobName(this ICompositeActivityBlueprint workflowBlueprint, string? activityId = null)
+        {
+            if (string.IsNullOrWhiteSpace(activityId) == false)
+            {
+                var activity = workflowBlueprint?.GetActivity(activityId);
+                if (activity is not null)
+                    return $"{workflowBlueprint?.DisplayName ?? workflowBlueprint?.Name ?? workflowBlueprint?.Id}->{activity?.DisplayName ?? activity?.Name ?? activity?.Id}";
+            }
+            if (workflowBlueprint is not null)
+                return $"{workflowBlueprint?.DisplayName ?? workflowBlueprint?.Name ?? workflowBlueprint?.Id}";
+            return null;
+        }
+
```

Comment:
Since this seems to be specific to generating a job name, perhaps we should move this extension method close to where it's used? Maybe somewhere in Elsa.Activities.Temporal.Common? Or is this used more broadly beyond the domain of scheduling and jobs?

Reply:
This is for creating a JobName for a blueprint, anything can use it to generate a job name. if I move it to the Elsa.Activities.Temporal.Common project, that will require the Elsa.Server.Hangfire project to have a dependency on a Elsa.Activities project. And that may not be desired by some.

Reply:
@sfmskywalker I rolled a null reference bugfix in this as well.

Reply:
When can this be merged?

Reply:
Will merge now.